patches/react-native-image-picker+5.1.0.patch

```diff
@@ -0,0 +1,134 @@
+diff --git a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java
+index 89b69a8..9345508 100644
+--- a/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java
++++ b/node_modules/react-native-image-picker/android/src/main/java/com/imagepicker/ImagePickerModuleImpl.java
+@@ -29,6 +29,121 @@ public class ImagePickerModuleImpl implements ActivityEventListener {
+     public static final int REQUEST_LAUNCH_VIDEO_CAPTURE = 13002;
+     public static final int REQUEST_LAUNCH_LIBRARY = 13003;
+
++    // Avoid image picker to show svg images on Android, to get the behaviour to be same as on iOS.
++    // Since, we can't exclude an image format, we instead allowlist all the possible image formats,
++    // excluding the 'image/svg+xml' mime type.
++    //
++    // Generated by merging from lists of image mimetypes supported on Android, and list of IANA media-types:
++    //   https://android.googlesource.com/platform/external/mime-support/+/main/mime.types#636
++    //   https://www.iana.org/assignments/media-types/media-types.xhtml#image
```

Comment:
Can we just use the Android one to be sure we are not adding something that was not added before